### DISCUSSION TITLE
After running Dcm4che Tool StowRS, nothing is happening.

### DISCUSSION THREAD

**Discussion by ghost** [dcm4che/dcm4che#927] [Q&A] [answered]
Hi, I am using dcm4che-tool-stowrs. When I run it, nothing happens. 
All the steps I followed, for running StowRS, are written below:

I have downloaded StowRS from the following link:
https://github.com/dcm4che/dcm4che/tree/master/dcm4che-tool/dcm4che-tool-stowrs

And, I am running one of the below commands:
mvn exec:java -D"exec.mainClass"="org.dcm4che3.tool.stowrs.StowRS" -D"exec.args"="--contentType image/jpeg -m StudyInstanceUID=1.3.12.2.1107.5.2.47.179239.30000021032203522820400000001 --url http://localhost:8082/dcm4chee-arc/aets/DCM4CHEE/rs/studies image.jpg"

mvn exec:java -D"exec.mainClass"="org.dcm4che3.tool.stowrs.StowRS" -D"exec.args"="--contentType image/jpeg -m PatientName=Arnold^Oscar --url http://localhost:8082/dcm4chee-arc/aets/DCM4CHEE/rs/studies image.jpg"

But, neither one of the commands is working since image is not being sent to DICOM AETitle. 
Along with that, I am getting an error:
ERROR stowrs.StowRS: Exception : resource:secondaryCaptureImageMetadata.xml

Please advice what can I do next.

**Comment by vrindanayak** [comment, marked as answer]
Answered in https://groups.google.com/g/dcm4che/c/3jLyLmEsREo
